src/main/java/org/broadinstitute/consent/http/service/EmailNotifierService.java

```diff
@@ -217,15 +217,15 @@ public void sendAdminFlaggedDarApproved(String darCode, List<User> admins, Map<U
         }
     }
 
-    public void sendNeedsPIApprovalMessage(Map<User, List<DataSet>> dataSetMap, Document access, Integer amountOfTime) throws MessagingException, IOException, TemplateException {
```

Comment:
I just commented out these methods because they are unused but need refactoring. Do we need to keep them for some reason? If we want to keep them then I will uncomment and refactor otherwise I'll just delete them

Reply:
We can remove them